launcher/ui/MainWindow.h

```diff
@@ -167,6 +167,10 @@ class MainWindow : public QMainWindow {
 
     void on_actionCreateInstanceShortcut_triggered();
 
+    void on_actionCreateInstanceShortcutDesktop_triggered();
```

Comment:
Can you add a default action for the original create shorcut item(the one that expands to actions)?

Reply:
I'm blind sorry.